recipes/vtk-cdat/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "2.10" %}
+{% set name = "vtk-cdat" %}
+
+package:
+    name: {{ name }}
+    version: 7.1.0.{{ version }}
```

Comment:
These jinja variables are declared at the top so that when you want to update you just change the version variable. Here, if the version needs to be updated it has to be changed in several places which defeats the purpose.